e2e/cypress/integration/team/teams_spec.js

```diff
@@ -182,4 +182,50 @@ describe('Teams Suite', () => {
         // * Ensure user is logged out
         cy.url({timeout: TIMEOUTS.LARGE}).should('include', 'login');
     });
+
+    it('TS14632 Team Icon should be Removed', () => {
+        cy.apiLogin('sysadmin');
+        cy.apiCreateTeam('test-team', 'Test Team').then(() => {
+            cy.visit('/');
+        });
+        cy.get('#sidebarHeaderDropdownButton').click();
+        cy.get('#teamSettings').click();
+
+        // 1. On a Team taht has a team icon
+        cy.get('#team_iconEdit').click();
+        cy.fileUpload('.section-max input', '../fixtures/mattermost-icon.png', 'image/png').trigger('change', {force: true});
```

Comment:
Please assign ID to input.
```suggestion
        cy.fileUpload('#inputSettingPicture', '../fixtures/mattermost-icon.png', 'image/png').trigger('change', {force: true});
```

Reply:
@saturninoabril Are we sure this element won't be used multiple times in any given part of the application? If so, that violates W3C/WCAG.

Reply:
Yes, I'm sure. Currently, It's being used in Account and Team Settings modal where such modal is not rendered at the same time (being unmounted when not in used). Same is true for other suggested new IDs.